controllers/authorino_controller.go

```diff
@@ -384,7 +384,7 @@ func (r *AuthorinoReconciler) buildAuthorinoArgs(authorino *api.Authorino) []str
 	if tracingServiceEndpoint := authorino.Spec.Tracing.Endpoint; tracingServiceEndpoint != "" {
 		args = append(args, fmt.Sprintf("--%s=%s", flagTracingServiceEndpoint, tracingServiceEndpoint))
 		for key, value := range authorino.Spec.Tracing.Tags {
-			args = append(args, fmt.Sprintf(`--%s="%s=%s"`, flagTracingServiceTag, key, value))
+			args = append(args, fmt.Sprintf(`--%s=%s=%s`, flagTracingServiceTag, key, value))
```

Comment:
Have you tested with tag values containing white spaces?

```yaml
spec:
  tracing:
    tags:
      tag-with-white-space: value with spaces
```

Maybe also a key containing white spaces:

```yaml
spec:
  tracing:
    tags:
      "tag key with spaces": some-value
```

I wonder if it could break the deployment.

Reply:
Yep tested now:

```yaml
spec:
   containers:
   - args:
     - --allow-superseding-host-subsets
     - --tracing-service-endpoint=rpc://jaeger-collector.istio-system.svc.cluster.local:4317
     - --tracing-service-insecure
     - --tracing-service-tag=random-key=value with spaces
     - --tracing-service-tag=test-key=test-value
```

![image](https://github.com/Kuadrant/authorino-operator/assets/6575004/914429dd-bd58-412f-a07e-083360b2ad72)